.github/workflows/e2e-nightly-master.yml

```diff
@@ -1,5 +1,4 @@
-# Runs randomly generated E2E testnets nightly.
-# Copy of earlier nightly test file, for debugging
+# Runs randomly generated E2E testnets nightly on master
 name: e2e-nightly2
```

Comment:
```suggestion
name: e2e-nightly-master
```
? easier to distinguish which one is master.

Reply:
Ah yes, thanks!